website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,455 @@
+
+      - <a name="v-server-service-enabled" href="#v-server-service-enabled">`enabled`</a> (`boolean: true`) -
+      Enables a service to allow other pods running in Kubernetes to communicate with the Vault server.
+
+  * <a name="v-server-datastorage" href="#v-server-datastorage">`dataStorage`</a> - This configures the volume used for storing Vault data when not using external storage such as Consul.
+
+      - <a name="v-server-datastorage-enabled" href="#v-server-datastorage-enabled">`enabled`</a> (`boolean: true`) -
+      Enables a persistent volume to be created for storing Vault data when not using an external storage service.
+
+      - <a name="v-server-datastorage-size" href="#v-server-datastorage-size">`size`</a> (`string: 10Gi`) -
+      Size of the volume to be created for Vault's data storage when not using an external storage service.
+
+      - <a name="v-server-datastorage-storageclass" href="#v-server-datastorage-storageclass">`storageClass`</a> (`string: null`) -
+      Name of the storage class to use when creating the data storage volume.
+
+      - <a name="v-server-datastorage-accessmode" href="#v-server-datastorage-accessmode">`accessMode`</a> (`string: ReadWriteOnce`) -
+      Type of access mode of the storage device.  See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes for more information.
+
+  * <a name="v-server-auditstorage" href="#v-server-auditstorage">`auditStorage`</a> - This configures the volume used for storing Vault's audit logs.  See https://www.vaultproject.io/docs/audit/index.html.
+
+      - <a name="v-server-auditstorage-enabled" href="#v-server-auditstorage-enabled">`enabled`</a> (`boolean: true`) -
+      Enables a persistent volume to be created for storing Vault's audit logs.
+
+      - <a name="v-server-auditstorage-size" href="#v-server-auditstorage-size">`size`</a> (`string: 10Gi`) -
+      Size of the volume to be created for Vault's audit logs.
+
+      - <a name="v-server-auditstorage-storageclass" href="#v-server-auditstorage-storageclass">`storageClass`</a> (`string: null`) -
+      Name of the storage class to use when creating the audit storage volume.
+
+      - <a name="v-server-auditstorage-accessmode" href="#v-server-auditstorage-accessmode">`accessMode`</a> (`string: ReadWriteOnce`) -
+      Type of access mode of the storage device.
+
+  * <a name="v-server-dev" href="#v-server-dev">`dev`</a> - This configures `dev` mode for the Vault server.
+
+      - <a name="v-server-dev-enabled" href="#v-server-dev-enabled">`enabled`</a> (`boolean: false`) -
+      Enables `dev` mode for the Vault server.  This mode is useful for experimenting with Vault without needing to unseal.
+
+        ~> **Security Warning:** Never, ever, ever run a "dev" mode server in production. It is insecure and will lose data on every restart (since it stores data in-memory). It is only made for development or experimentation.
```

Comment:
Maybe link to a "production ready" section with a reference architecture or check list for the making this production ready?